From 8d801108bceb4c1f02da07a98cc5742f14578194 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Wed, 18 Oct 2023 00:04:26 +0200
Subject: [PATCH] Fix compile error in some setups

Move namespace'd libintl.h include further down so the very first
include of various system headers isn't namespaced as well.

Fixes very weird compile errors on Alpine Linux.
---
 indicator/authhandler.cpp   | 8 ++++----
 indicator/ussdindicator.cpp | 8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/indicator/authhandler.cpp b/indicator/authhandler.cpp
index 5847d862..366ecdb1 100644
--- a/indicator/authhandler.cpp
+++ b/indicator/authhandler.cpp
@@ -19,16 +19,16 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-namespace C {
-#include <libintl.h>
-}
-
 #include <QDebug>
 #include <libnotify/notify.h>
 #include "applicationutils.h"
 #include "authhandler.h"
 #include "telepathyhelper.h"
 
+namespace C {
+#include <libintl.h>
+}
+
 // FIXME: port to libqmenumodel if possible
 AuthHandler::AuthHandler(QObject *parent)
 : QObject(parent),
diff --git a/indicator/ussdindicator.cpp b/indicator/ussdindicator.cpp
index 3854b195..4375f7c3 100644
--- a/indicator/ussdindicator.cpp
+++ b/indicator/ussdindicator.cpp
@@ -19,10 +19,6 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-namespace C {
-#include <libintl.h>
-}
-
 #include <QDebug>
 #include <libnotify/notify.h>
 #include "ringtone.h"
@@ -30,6 +26,10 @@ namespace C {
 #include "ofonoaccountentry.h"
 #include "telepathyhelper.h"
 
+namespace C {
+#include <libintl.h>
+}
+
 USSDIndicator::USSDIndicator(QObject *parent)
 : QObject(parent),
   m_menuRequest("ussd", true),
-- 
2.42.0

